cc3d/player5/Utilities/__init__.py

```diff
@@ -3,4 +3,4 @@
 # __all__ = ['QVTKRenderWidget', 'QVTKRenderWindowInteractor']
 __all__ = ['QVTKRenderWidget']
 
-from . utils import *
\ No newline at end of file
+from cc3d.player5.Utilities.utils import *
```

Comment:
minor comment but it would not be bad to eliminate `import  *` statements in favor of importing actual names used in a given module. this is more of a cleanup task so no need to rush now but something we can work as a side projects

Reply:
I always prefer to not use `*`. Makes my debugger too cluttered. I like the idea of doing what's suggested here throughout the codebase as a running cleanup task.